src/app/search/search-filters.ts

```diff
@@ -1326,6 +1271,7 @@ function searchFilters(
           item.sockets.sockets.some((socket) => {
             return !!(
               socket.plug &&
+              // these hashes are all forms of "empty socket"
               ![2323986101, 2600899007, 1835369552, 3851138800, 791435474].includes(
```

Comment:
It'd be nice to pull this into a named constant.

Reply:
on it

Reply:
it think these all have the following `plug.onActionRecreateSelf` = false

which normal mods do not

Reply:
or even `plug.insertionMaterialRequirementHash === 0`

Reply:
@delphiactual on my list. gonna test & implement it after this, so i can stop battling merge conflicts